src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,184 @@
+.. _Read_Write_Options:
+
+Database Read and Write Options
+-------------------------------
+Several database plugins have options that effect reading and/or writing with that
+format. These are described in this section. Alternatively, in some cases, the
+behavior of a database plugin may be effected by enviornment variables.
+
+Chombo
+~~~~~~
+
+Exodus
+~~~~~~
+   
+
+Detect Compound Variables
```

Comment:
This is a cool feature!

Reply:
Thanks. Now that I think about it...this really should be something we offer *above* all the plugins, instead of coding it in individual plugins.